transports/webrtc/Cargo.toml

```
@@ -0,0 +1,43 @@
+[package]
+name = "libp2p-webrtc"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "WebRTC transport for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = ""
```

```suggestion
license = "MIT"
```

Are you OK with contributing this under the MIT license?

---

Sure